ApplicationInsights/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -271,12 +276,16 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
         this.core.track(telemetryItem);
     }
 
+    /**
+     * Send browser performance metrics.
+     * @param pageViewPerformance
+     * @param customProperties
```

Comment:
wondering if we should mark these as unused in the SKUs in comments

Reply:
actually we can just delete from SKU since its external facing